<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&lt;big>&lt;b>Active Filters&lt;/b>&lt;/big></string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&lt;big>&lt;b>Available Filters&lt;/b>&lt;/big></string>
     </property>
     <property name="textFormat" >
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonAdd" >
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPreview" >
       <property name="text" >
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose" >
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonConfigure" >
       <property name="text" >
        <string>Configure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemove" >
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonPartial" >
       <property name="text" >
        <string>Partial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonUp" >
       <property name="text" >
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonDown" >
       <property name="text" >
        <string>Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTabWidget" name="tabWidgetSubtitles" >
     <property name="tabPosition" >
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>5</number>
     </property>
     <property name="elideMode" >
      <enum>Qt::ElideLeft</enum>
     </property>
     <widget class="QWidget" name="Transform" >
      <attribute name="title" >
       <string>Transfom</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="filter.qrc" >:/new/prefix1/icons/1.png</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QListWidget" name="listWidgetTransform" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Interlacing" >
      <attribute name="title" >
       <string>Interlacing</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="filter.qrc" >:/new/prefix1/icons/2.png</iconset>
      </attribute>
      <attribute name="toolTip" >
       <string>Interlacing</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QListWidget" name="listWidgetInterlacing" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Colors" >
      <attribute name="title" >
       <string>Colors</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="filter.qrc" >:/new/prefix1/icons/4.png</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QListWidget" name="listWidgetColors" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Noise" >
      <attribute name="title" >
       <string>Noise</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="filter.qrc" >:/new/prefix1/icons/5.png</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QListWidget" name="listWidgetNoise" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Subtitles" >
      <attribute name="title" >
       <string>Sharpness</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="filter.qrc" >:/new/prefix1/icons/3.png</iconset>
      </attribute>
      <attribute name="toolTip" >
       <string>Sharpness</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QListWidget" name="listWidgetSharpness" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Sharpness" >
      <attribute name="title" >
       <string>Subtitles</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="filter.qrc" >:/new/prefix1/icons/7.png</iconset>
      </attribute>
      <attribute name="toolTip" >
       <string>Subtitles</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QListWidget" name="listWidgetSubtitles" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Misc" >
      <attribute name="title" >
       <string>Misc</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="filter.qrc" >:/new/prefix1/icons/6.png</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QListWidget" name="listWidgetMisc" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QListWidget" name="listWidgetActive" />
   </item>
  </layout>
 </widget>
 <resources>
  <include location="filter.qrc" />
 </resources>
 <connections/>
</ui>
